plugins/inventory/cobbler.py

```diff
@@ -210,7 +218,7 @@ def parse(self, inventory, loader, path, cache=True):
                 while i < len(profile_elements) - 1:
                     profile_group = '-'.join(profile_elements[0:i + 1])
                     profile_group_child = '-'.join(profile_elements[0:i + 2])
-                    if profile_group in self.exclude_profiles:
+                    if self._exclude_profile(profile_group):
                         self.display.vvvv('Excluding profile %s\n' % profile_group)
                         break
```

Comment:
Should this actually be a `break`, or a `continue`?

Reply:
Here we are creating a hierarchy of profiles out of a profile name like "redhat-8-x86_64":  redhat -> redhat-8 -> redhat-8-x86_64.  If we exclude "redhat-8" we want to exclude the child profiles as well, so break is what we want.